testgrid/config/config.proto

```diff
@@ -2,6 +2,25 @@ syntax = "proto3";
 
 // Protocol buffer for configuring testgrid.k8s.io
 
+// Specifies the test name.
+message TestNameConfig {
+  // Specifies name elements to be selected from configuration values
+  message NameElement {
+    reserved 1, 3, 4;
+
+    // Configuration value to use
```

Comment:
Please explain this better. I have no idea how I can use this and what this is connect to. What is target_config? What name_elements do I have to choose from?